### PR TITLE
Add the release note for Nodejs client 1.8.0

### PR DESCRIPTION
**Pull request by RobertIndie**
Preview:
image
image

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]

File: release-notes/versioned/pulsar-client-node-1.8.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: Pulsar Client Node 1.8.0
+sidebar_label: Pulsar Client Node 1.8.0
+---
+
+## What's Changed
+* [feat] Add chunking support by @RobertIndie in https://github.com/apache/pulsar-client-node/pull/234
+* [feat] Napi support package c++ static binary by @shibd in https://github.com/apache/pulsar-client-node/pull/235
+* [feat] Support windows build napi of static link c++ lib by @shibd in https://github.com/apache/pulsar-client-node/pull/238
+* [refactor] Modify the README to apply the new installation method by @shibd in https://github.com/apache/pulsar-client-node/pull/239
+* [feat] Add support for seek by message id by @RobertIndie in https://github.com/apache/pulsar-client-node/pull/241
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-client-node-1.8.0.md:12]:
```suggestion
* [feat] Add support for seeking by message id by @RobertIndie in https://github.com/apache/pulsar-client-node/pull/241
```

Comment (Anonymitaet) [inline release-notes/versioned/pulsar-client-node-1.8.0.md:12]:
```suggestion
* [feat] Add support for seeking by message ID by @RobertIndie in https://github.com/apache/pulsar-client-node/pull/241
```

File: release-notes/versioned/pulsar-client-node-1.8.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* [refactor] Fetch C++ client official release by @shibd in https://github.com/apache/pulsar-client-node/pull/244
+* [feat] Support consumer seek by timestamp and reader seek by @RobertIndie in https://github.com/apache/pulsar-client-node/pull/247
+* [feat] Supoort prebuild macos-arm64 arch binaries by @shibd in https://github.com/apache/pulsar-client-node/pull/250
+* [fix] Upgrade the version of OpenSSL that compiles MacOS cpp client dependencies by @shibd in https://github.com/apache/pulsar-client-node/pull/251
```
Comment (Anonymitaet) [inline release-notes/versioned/pulsar-client-node-1.8.0.md:16]:
```suggestion
* [fix] Upgrade the version of OpenSSL that compiles MacOS CPP client dependencies by @shibd in https://github.com/apache/pulsar-client-node/pull/251
```

File: release-notes/versioned/pulsar-client-node-1.8.0.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* [improve] Use async C function in the Consumer#receive method by @equanz in https://github.com/apache/pulsar-client-node/pull/252
+* [feat] Add example for custom logger and LogLevel#toString by @RobertIndie in https://github.com/apache/pulsar-client-node/pull/253
+* [fix] Fix windows default download cpp client failed by @shibd in https://github.com/apache/pulsar-client-node/pull/257
```
Comment (Anonymitaet) [inline release-notes/versioned/pulsar-client-node-1.8.0.md:19]: same

File: release-notes/versioned/pulsar-client-node-1.8.0.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+* [refactor] Remove useless dependencies. by @shibd in https://github.com/apache/pulsar-client-node/pull/258
+* [refactor] Refactor the cpp client version controlling in the build script by @RobertIndie in https://github.com/apache/pulsar-client-node/pull/259
```
Comment (Anonymitaet) [inline release-notes/versioned/pulsar-client-node-1.8.0.md:21]: same

**Review by Anonymitaet — COMMENTED** [review]
Do we need to add the contributors for this release at the end (to encourage more contributions)?

**Review by momo-jun — APPROVED** [review]

**Comment by RobertIndie** [comment]
@Anonymitaet From the previous discussion, we add the `New contributors` section for each release. But since there is no new contributors for this release, so I remove this section. And there is a contributors list in the GitHub release, for example: https://github.com/apache/pulsar-client-cpp/releases/tag/v3.1.0.

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**